apps/dotcom/src/__snapshots__/routes.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`the_routes 1`] = `
 [
   {
     "reactRouterPattern": "/",
-    "vercelRouterPattern": "^//?$",
+    "vercelRouterPattern": "^/?$",
```

Comment:
wait why did this need to change? I can't imagine vercel was matching against the empty string for the root path.

Reply:
yeah that's what i said above in the comment:
> ugh, our SPA routes were wrong actually (so this was working on other paths but not on the / base path!). the SPA routes are generated from the mapping in routes.test.tsx.snap which i just realized was wrong for the index file 🤦 (it was set to "^//?$" instead of "^/?$")

vercel was _not_ matching on this - this was broken :-/

Reply:
double-forward slash isn't an escape sequence - `\\` double _back_slash would be though 🙃

Reply:
it's not trying to escape anything, `"^//?$"` matches a '/' at the start of the string, followed by an optional '/', then nothing more. so it matches the paths `/` and `//`, so it should have been fine when you go to `tldraw.com` unless vercel was matching against the empty string which seems quite unlikely to me? i guess stranger things have been known to happen...

Reply:
ach! right - totally read that regex wrong, my bad.

ok, actually something is still wrong here then. with my latest push, it's still not recognizing the main `/` path, only the other paths 🤔 it actually _might_ be the empty path?? i'm going to push up and try.

Reply:
nope. that didn't work - going to keep trying other things - bit of a black box, not sure what it's seeing.
going to try `/index.html` next...

Reply:
the latest change did it! thx for rubber ducking with me @ds300